install_utils.py

```diff
@@ -0,0 +1,172 @@
+import sys
+import os
+import subprocess
+
+
+# input/output utils
+def ask(question, default="yes"):
+    """Ask a yes/no question via raw_input() and return their answer.
```

Comment:
I usually have the first line of the docstring under the `"""`, as you have below. But that's personal preference